ballerina/tests/tests.bal

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+
+configurable boolean IS_LIVE_SERVER = ?;
+configurable string MAILCHIMP_USERNAME = ?;
+configurable string MAILCHIMP_API_KEY = ?;
+configurable string MAILCHIMP_BASE_URL = ?;
+
+Client mailchimp = check new ({
+    auth: {
+        username: MAILCHIMP_USERNAME,
+        password: MAILCHIMP_API_KEY
+    }
+}, IS_LIVE_SERVER ? MAILCHIMP_BASE_URL : "http://localhost:9090");
+
+@test:Config
+function listAutomations() returns error? {
+    InlineResponse2005 response = check mailchimp->/automations();
+    test:assertNotEquals(response, ());
```

Comment:
I guess this assertion doesn't add a value since this statement always returns true. (Probably the IDE should report this). Instead of checking whether the `response` is not `()`, can we check a property of the response record? Check similar places.